app/controllers/shared/api.php

```diff
@@ -165,6 +165,15 @@
             $queueForUsage
                 ->addMetric(METRIC_FUNCTIONS, $value); // per project
 
+            if ($event === Database::EVENT_DOCUMENT_DELETE) {
+                $queueForUsage
+                    ->addReduce($document);
+            }
+            break;
+        case $document->getCollection() === 'sites':
+            $queueForUsage
+                ->addMetric(METRIC_SITES, $value); // per project
```

Comment:
Keep in mind we'll be deprecating this event listener and start counting resources with a DB count method instead of during runtime.

Reply:
Damodar mentioned it's already implemented in statsResources too, so it's okay.
He also takes lead in removing some of those later on, and is aware of sites addition.

Marking this as close as we have proper vision on it, and doesnt require immidiate actioning